config.toml

```diff
@@ -85,8 +85,8 @@ project_home = "https://openservicemesh.io/"
   version = "latest"
   url = "https://docs.openservicemesh.io/"
 [[params.versions]]
-  version = "v0.8"
-  url = "https://release-v0-8.docs.openservicemesh.io/"
+  version = "v0.9"
+  url = "https://release-v0-9.docs.openservicemesh.io/"
```

Comment:
Ah, we want to keep both (perhaps in reverse order?) as there will be customers who want the v0.8.x docs for some time. So when you add v.09, also keep a v.08 one too. You can have an arbitrary number of `version = "whatever` choices, each with a url.

Reply:
thank you @bridgetkromhout  for the feedback.

fixec

Reply:
Oops, just realized - can you remove this change for now, as we need to ensure the docs are published and _then_ link to them in a second PR?